font-src/glyphs/marks/above.ptl

```diff
@@ -523,6 +535,26 @@ glyph-block Mark-Above : begin
 			flat (m2 - markStress) aboveMarkBot [widths markFine markFine]
 			curl (m2 + markExtend) aboveMarkTop [widths markStress markStress]
 
+	create-glyph 'tripleAcuteAbove' 0x1ACB : glyph-proc
+		set-width 0
+		include : StdAnchors.impl 'above' 0 1.5
+
+		local fine : markFine * 2 / 3
+		local stress : markStress * 2 / 3
```

Comment:
I think here we could use `[AdviceStroke]` with a larger crowdedness. Maybe this:

```
local fine : 0.5 * [AdviceStroke 6]
local stress : 1.2 * fine
```

Adjust this number "6" inside `[AdviceStroke]` to get a good visual under both thin, regular and heavy.